ui/src/components/Material/Tabs/styles.css

```diff
@@ -1,3 +1,7 @@
 .tabs .pointer {
   background-color: rgb(245,171,53);
 }
+
+.tab {
+  overflow: visible;
```

Comment:
react-toolbox 2 has breaking changes. 
https://github.com/react-toolbox/react-toolbox/commit/cd6a130bac655bbd52a0f3872d09dcfce2a898a4
This is one of them.